api/Performance.json

```diff
@@ -51,12 +51,26 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance/clearMarks",
           "support": {
-            "chrome": {
-              "version_added": "43"
-            },
-            "chrome_android": {
-              "version_added": "46"
-            },
+            "chrome": [
+              {
+                "version_added": "29"
+              },
+              {
+                "version_added": "25",
+                "version_removed": "29",
+                "prefix": "-webkit-"
```

Comment:
This edit is contrary to [how the prefix is actually written in IDL and code](https://chromium.googlesource.com/chromium/src/+/aa0ed75290c068427f90ede29a6efaf02e6e6522%5E%21/#F4). What's the official position on that?

Reply:
Oh I think I see what you mean. So maybe like this instead?

```suggestion
                "alternative_name": "webkitClearMarks"
```

Reply:
The issue is that in IDL files prefixes are as I've done them here (no dashes), but in CSS they are exactly as you suggest (with dashes). 

There are a bunch that need be corrected.

Reply:
https://cs.chromium.org/chromium/src/third_party/blink/web_tests/webexposed/css-property-listing-expected.txt

Reply:
See #3670 for the complete explanation.

Reply:
Sorry, I think one or both of us has something turned around.

Here's what I'm seeing in your original diff: `"prefix": "-webkit-"`

I thought this is what I suggested: `"prefix": "webkit"` (which is, itself, not quite right because of the case of the first character, check out #3488 for more of this) and then (in a #3488-friendly way) `"alternative_name": "webkitClearMarks"`

Are you saying there are more APIs with `"prefix": "-webkit-"`? Or is this a question about whether CSS should be documented with dashes or not?

Reply:
I apologize. I have been submitting lousy work lately and you've born the brunt of it. I will do better.

Reply:
No worries. We seem to do a pretty good job of getting it right _before_ we merge, so I can't complain. 😄